server/repository/src/db_diesel/changelog/README.md

```diff
@@ -0,0 +1,42 @@
+# Changelog
+
```

Comment:
Think an intro sentence would be good (or move last sentence to the top):

The changelog records database change for selected tables, i.e. if a row has been upserted or deleted.

Reply:
Thanks: f98a4420fc3ec99d5c5bb13e1676d4f72a9c8717